about.html

```diff
@@ -0,0 +1,157 @@
+      integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <script src="index.js" defer></script>
+    <title>About</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="banner">
+        <ul class="banner-list">
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My page</a></li>
+          <li><a href="#">logout</a></li>
+        </ul>
+      </div>
+      <nav class="navbar">
+        <div class="navbar-container">
+          <a href="./index.html"
+            ><img src="./images/UN_emblem_blue.png" alt="" class="logo"
+          /></a>
+
+          <div class="hamburger"><i class="fa-solid fa-bars"></i></div>
+          <ul class="menu-list">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Join</a></li>
+            <li class="cc-campaign"><a href="#">CC Campaign</a></li>
+          </ul>
+          <ul class="mobile-menu">
+            <div class="closebutton"><i class="fa-solid fa-x"></i></div>
+            <li><a href="./index.html">Home</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">News</a></li>
+            <li><a href="#">Join</a></li>
+            <li class="cc-campaign"><a href="#">CC Campaign</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <section class="about">
+      <div class="aboutcontainer">
+        <h1 class="h1">
+          WOMENS RIGHTS <br />
+          GLOBAL SUMIT 2015
+        </h1>
+
+        <div class="topic1container">
+          <p class="topic1">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the women in leadership from
+            different country every two years. This year, we hope to expand our
+            invitation list including organizations who want to work with us on
+            shared projects that advance the cause of woman empowerment. So if
+            you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, wikipedians, Open knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant community
+            together.
+          </p>
+        </div>
```

Comment:
- [ ] Please add the appropriate padding to this section. 

<img width="356" alt="image" src="https://github.com/enocol/capstone-project/assets/84156714/efef1f5a-8cb4-4637-88b3-1dfabf7b8aa7">

Reply:
Padding applied